database/seeder/DatabaseSeeder.php

```diff
@@ -6,194 +6,201 @@
 use Anomaly\NavigationModule\Link\LinkModel;
 use Anomaly\NavigationModule\Menu\Contract\MenuRepositoryInterface;
 use Anomaly\Streams\Platform\Entry\EntryRepository;
-use Anomaly\Streams\Platform\Model\Users\UsersUsersEntryModel;
 use Anomaly\UrlLinkTypeExtension\UrlLinkTypeModel;
 use Anomaly\UsersModule\Role\Contract\RoleRepositoryInterface;
-use Anomaly\UsersModule\User\Contract\UserInterface;
 use Anomaly\UsersModule\User\Contract\UserRepositoryInterface;
 use Anomaly\UsersModule\User\UserActivator;
+use Illuminate\Console\Command;
 use Illuminate\Database\Seeder;
 use Anomaly\DashboardModule\Widget\Contract\WidgetRepositoryInterface;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\Facades\Artisan;
 use Illuminate\Support\Facades\DB;
+use widgetSeeder;
```

Comment:
Class names should start with a capital letter 😉

Reply:
It wasn't a file I created. But I'm fixing it. ;)